luigi_pipeline/seqr_loading.py

```diff
@@ -148,6 +124,30 @@ def annotate_old_and_split_multi_hts(self, mt):
         # Named `locus_old` instead of `old_locus` because split_multi_hts drops `old_locus`.
         return hl.split_multi_hts(mt.annotate_rows(locus_old=mt.locus, alleles_old=mt.alleles))
 
+    @staticmethod
+    def contig_check(mt, standard_contigs, threshold):
```

Comment:
I moved this method onto the class so that the mocking looked similar to the existing mocking.